pkg/services/store/k8saccess/dashboard_service.go

```diff
@@ -0,0 +1,93 @@
+package k8saccess
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/dashboards"
+	"github.com/grafana/grafana/pkg/services/store/entity"
+)
+
+type k8sDashboardService struct {
+	orig  dashboards.DashboardService
+	store entity.EntityStoreServer
```

Comment:
```go
    if features.IsEnabled(featuremgmt.FlagK8sDashboards) {
		if k8s.GetSystemClient() == nil {
			panic("k8s dashboards requires the k8s client registered")
		}
		return k8saccess.NewDashboardService(svc, store)
	}
	return svc
```
i'm not following how either the original dashboard store or the entity store will be used here. based on this feature flag i was assuming this was writing to k8s, and then the k8s controller would write to the original store?

Reply:
yes -- we just need to intercept the write/modify/delete paths, the rest still has to be called from the original service